angular-workspace/projects/ni/nimble-angular/src/directives/rich-text-viewer/tests/nimble-rich-text-viewer.directive.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component, ViewChild } from '@angular/core';
+import { NimbleRichTextViewerModule } from '../nimble-rich-text-viewer.module';
+import { NimbleRichTextViewerDirective } from '../nimble-rich-text-viewer.directive';
+
+describe('Nimble Rich Text Viewer', () => {
```

Comment:
If you take a look at [the spec for some other directives](https://github.com/ni/nimble/blob/main/angular-workspace/projects/ni/nimble-angular/src/directives/button/tests/nimble-button.directive.spec.ts) you can see the pattern we've established for testing them. It includes testing each property in 3 different `describe` blocks for 3 different scenarios. Could you try to copy that pattern for this component?

Reply:
Yes, I have written the tests in [this branch](https://github.com/ni/nimble/blob/users/vivin/update-angular-directive-for-rich-text-viewer/angular-workspace/projects/ni/nimble-angular/rich-text-viewer/tests/nimble-rich-text-viewer.directive.spec.ts) for the API and will create a separate PR once the https://github.com/ni/nimble/pull/1335 is completed

We have decided to raise this PR now to include `nimble-rich-text-viewer` in the wrapper component planned for the comments feature in SLE and create separate PRs whenever we update the component. This will allow us to begin working with the actual `nimble-rich-text-viewer` importing from `@ni/nimble-angular`.

Reply:
Excellent, I'm glad you've got those tests written already. Our general preference is to submit the tests with their corresponding code since it makes it easier to track that no test coverage gets missed. So for this PR that would mean either:
1. copy the full contents of `nimble-rich-text-viewer.directive.spec.ts` from your other branch to this one
2. remove the `markdownValue` and `fitToContent` properties from the directive in this PR and add them in the follow-on PR

I may be missing something that makes those options challenging but if it's as simple as moving some files between branches then let's default to keeping the code and tests in sync at all times.

Reply:
Sure, I have removed the properties and had only the relevant test cases covered for bringing up the angular code. I will update it in the follow on PRs.